azure-ipam/ipam.go

```diff
@@ -72,7 +73,7 @@ func (p *IPAMPlugin) CmdAdd(args *cniSkel.CmdArgs) error {
 	p.logger.Debug("Making request to CNS")
 	// if this fails, the caller plugin should execute again with cmdDel before returning error.
 	// https://www.cni.dev/docs/spec/#delegated-plugin-execution-procedure
-	resp, err := p.cnsClient.RequestIPAddress(context.TODO(), req)
+	resp, err := p.cnsClient.RequestIPs(context.TODO(), req)
```

Comment:
we need to call the old API if this fails

Reply:
The 404 check is below